corehq/apps/app_manager/views/utils.py

```diff
@@ -411,3 +413,18 @@ def clear_xmlns_app_id_cache(domain):
 
 def form_has_submissions(domain, app_id, xmlns):
     return FormES().domain(domain).app([app_id]).xmlns([xmlns]).count() != 0
+
+
+def get_multimedia_sizes_for_build(domain, build_id, build_profile_id=None):
+    build = get_app_cached(domain, build_id)
+    assert build.copy_of, _("Size calculation available only for builds")
+    build_profile = build.build_profiles[build_profile_id] if build_profile_id else None
+    # path: HQMediaMapItem object
+    multimedia_map_for_build = build.multimedia_map_for_build(build_profile=build_profile)
+    # path: CommCareMultimedia object
+    media_objects = dict(build.get_media_objects(multimedia_map=multimedia_map_for_build))
```

Comment:
Should this be an `OrderedDict` to guarantee that "Total" appears last?

Reply:
Just found this,
https://mail.python.org/pipermail/python-dev/2017-December/151283.html
So i guess this should be fine if above is true :)

Reply:
aha, I forgot that's a py3 change